start.sh

```diff
@@ -63,6 +63,21 @@ https://mail.{\$NC_DOMAIN}:443 {
 CADDY
 fi
 
+if [ -n "$(dig A +short nextcloud-aio-jellyfin)" ] && ! grep -q nextcloud-aio-jellyfin /Caddyfile; then
+    cat << CADDY >> /Caddyfile
+https://jellyfin.{\$NC_DOMAIN}:443 {
+    reverse_proxy nextcloud-aio-jellyfin:8096
```

Comment:
Adding the geofilter herr makes sense as well I'd say?
```suggestion
    # import GEOFILTER
    reverse_proxy nextcloud-aio-jellyfin:8096
```

Reply:
I just didn't get the point of adding this, because it is a comment, right ?
Is it to allow people to set the GEOFILTER just by removing the comment ?